Blob[6] blobs